ADA/ada.qmd

```diff
@@ -21,55 +21,124 @@ It brings together:
+
+#### Writing and running code
+
+In Databricks, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+Scripts (e.g. R scripts) can be created and saved in your Workspace and run via the Databricks interface. 
+
+There is detailed guidance on using the Databricks SQL editor, which is similar to SSMS, on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-sql-in-databricks-sql-editor).
+
+One benefit of Databricks is the use of workflows to run your code. Workflows allow you to automate the running of single or multiple scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+---
+
+## What this means for existing code
 
 ---
 
-## Databricks code templates
+### SQL scripts
+
+If you currently use data and scripts stored in SSMS for your statistics publication pipeline, there will be small changes to your production pipelines that vary depending on your current process. 
+
+Your SQL code will need to be migrated over to Databricks to allow it to read from the Databricks SQL Warehouse and will require some minor changes to syntax as SSMS uses T-SQL and Databricks uses Spark SQL. The ADA team are currently developing a tool to allow you to easily convert T-SQL queries to Spark SQL syntax. 
+
+You can write new SQL code in the Databricks SQL Editor, which is very similar to SSMS. 
+
+### R scripts in RStudio
 
-See examples and code snippets in R, SQL and Python in these [Databricks code template notebooks](https://adb-6882499576863257.17.azuredatabricks.net/browse/folders/3070168888967919?o=6882499576863257#)
+::: {.callout-important}
+If you have an existing RAP process and pipeline set up using RStudio and SSMS, there is no current expectation for you to migrate your existing R code or scripts from RStudio to Databricks notebooks. However, you will need to migrate SQL code over to Databricks in order to maintain your publication pipelines with the new platform. 
+:::
 
-# What ADA means for statistics publications
+Your R code can remain in RStudio and all existing R processes can still be run there. Code that points towards SQL scripts will need to be redirected to Databricks rather than SSMS. In order to use RStudio with a Databricks SQL warehouse, you will need to manually set up a connection. You can find more information about that on our [setup guidance page](/ADA/databricks_rstudio_sql_warehouse.html). 
```

Comment:
I want fireworks and confetti and sirens on that first sentence - maybe move the important box to underneath this paragraph?

Reply:
ah man I had it here and moved it 😆